### PR TITLE
Redesign homepage & added issue report feature

### PR DESCRIPTION
**Pull request by DesmondTo**
1. Redesign homepage
   - Divided homepage into 3 sections: main content, advantages of the website, issue report section
2. At the bottom of the homepage, there exists a issue report form

Problem:
The issue report can only be created in the database(after the user click submit) if the user is login as admin, which is the same issue as the "login to book issue"

### CODE REVIEW

File: app/angular/AngularView/src/app/home/home.component.html [hunk 1]
```diff
@@ -1,18 +1,126 @@
...
-            <a id="but-2" mat-button mat-stroked-button routerLink="../edit" routerLinkActivate = "active">
-                Make a Booking
-            </a>
+        <!-- section 1 of the homepage: The main contents and buttons -->
+        <div class="sec1">
+            <div class="main">
+                <div class="left">
+                    <div class="main_caption">
+                        <h1>Booking has never been so <span id="EASY">EASY</span></h1>
+                        <p>We, commIT technical cell strive to cater you the best experience
```
Comment (JQChong) [inline app/angular/AngularView/src/app/home/home.component.html:11]: this text needs to be changed, but that's an issue for later

File: app/django/meridien/issues/views.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from issues.models import Issue
+from issues.serializers import IssueSerializer
+
+import sys
+sys.path.append('../')
+from meridien import views_template
+
+# Create your views here.
+@api_view(['GET', 'POST', 'DELETE'])
+@permission_classes([IsAuthenticated])
```
Comment (JQChong) [inline app/django/meridien/issues/views.py:18]: no wonder lolololololol

File: app/django/meridien/issues/views.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+@csrf_exempt
+def issue_list(request):
+    response = views_template.obj_list(request, Issue, IssueSerializer)
+    return response
+
+@api_view(['GET', 'PUT', 'DELETE'])
+@permission_classes([IsAuthenticated])
```
Comment (JQChong) [inline app/django/meridien/issues/views.py:25]: same here... but this one u need to protect so that ppl can't anyhow edit the reviews

**Review by JQChong — CHANGES_REQUESTED** [review]
for the image files pls put them into the assets folder... dun make the file directories even messier when it's already rather messy...

File: app/angular/AngularView/src/app/home/home.component.html (hunk 1 above)
Comment (DesmondTo) [reply app/angular/AngularView/src/app/home/home.component.html]: Text changed. Please check if it is appropriate. Thank you.

File: app/django/meridien/issues/views.py (hunk 2 above)
Comment (DesmondTo) [reply app/django/meridien/issues/views.py]: Changed IsAuthenticated to AllowAny. No other changes made.

File: app/angular/AngularView/src/app/home/home.component.html (hunk 1 above)
Comment (DesmondTo) [reply app/angular/AngularView/src/app/home/home.component.html]: All the additional files are moved into assets file.

File: app/django/meridien/issues/views.py [hunk 4]
```diff
@@ -15,9 +17,10 @@
 
 # Create your views here.
 @api_view(['GET', 'POST', 'DELETE'])
-@permission_classes([IsAuthenticated])
+@permission_classes([AllowAny])
 @csrf_exempt
 def issue_list(request):
+    permission_classes = (AllowAny,)
```
Comment (JQChong) [inline app/django/meridien/issues/views.py:23]: separate this into two views, the last thing i want is ppl to fool around with the api and then we get screwed

**Review by JQChong — COMMENTED** [review]
strangely enuf the smooth scrolling thing causes two scroll bars to appear in my browser... not sure if this is due to microsoft edge or what, idk lol

**Pull request merged**